content/docs/intro/concepts/programming-model.md

```diff
@@ -634,9 +747,13 @@ record = aws.route53.Record("validation",
 // See https://github.com/pulumi/pulumi/issues/1614.
 ```
 
+```csharp
+// Helpers for accessing properties are not yet available in .NET.
```

Comment:
Link to work item?

Reply:
i don't think we have a work item here.  it's literally not something we necessarily know can even be supported in the .net world.  it fundamentally makes use of dynamism in JS and the ability for TS to repesent htat.